contracts/vaults/TreasuryVault/RibbonTreasuryVault.sol

```diff
@@ -797,25 +752,22 @@ contract RibbonTreasuryVault is
 
         address recipient = feeRecipient;
         uint256 mintShares;
-        uint256 performanceFeeInAsset;
-        uint256 totalVaultFee;
+        uint256 managementFeeInAsset;
         {
             uint256 newPricePerShare;
             (
                 lockedBalance,
                 queuedWithdrawAmount,
                 newPricePerShare,
                 mintShares,
-                performanceFeeInAsset,
-                totalVaultFee
-            ) = VaultLifecycle.rollover(
+                managementFeeInAsset
+            ) = VaultLifecycleTreasury.rollover(
                 vaultState,
-                VaultLifecycle.RolloverParams(
+                VaultLifecycleTreasury.RolloverParams(
```

Comment:
Adding on to this: https://github.com/ribbon-finance/ribbon-v2/pull/185/files#r767468586

Since we access the `uint currentRound = vaultState.round;`, we can do:

```solidity
...
lastQueuedWithdrawAmount,
currentRound != 1 ? managementFee : 0)
```